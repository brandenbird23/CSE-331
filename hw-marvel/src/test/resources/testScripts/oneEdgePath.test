# Tests a path with only one edge between nodes

LoadGraph nfl nflPlayers.csv

FindPath nfl TOM-BRADY MIKE-EVANS