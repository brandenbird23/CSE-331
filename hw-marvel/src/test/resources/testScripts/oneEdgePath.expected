# Tests a path with only one edge between nodes

loaded graph nfl

path from TOM-BRADY to MIKE-EVANS:
TOM-BRADY to MIKE-EVANS via BUCS